android/sdl_android/src/androidTest/java/com/smartdevicelink/test/proxy/SystemCapabilityManagerTests.java

```diff
@@ -490,6 +490,94 @@ public void testAddOnSystemCapabilityListenerWithSubscriptionsNotSupportedAndCap
 		verify(internalInterface, times(1)).sendRPC(any(GetSystemCapability.class));
 	}
 
+	public void testAddOnSystemCapabilityListenerThenGetCapability() {
+		SdlMsgVersion sdlMsgVersion = new SdlMsgVersion(5, 0); // This version doesn't support capability subscriptions
+		sdlMsgVersion.setPatchVersion(0);
+		ISdl internalInterface = mock(ISdl.class);
+		when(internalInterface.getSdlMsgVersion()).thenReturn(sdlMsgVersion);
+		SystemCapabilityManager scm = new SystemCapabilityManager(internalInterface);
+		scm.setCapability(SystemCapabilityType.VIDEO_STREAMING, videoStreamingCapability);
+
+
+		// Add listener1
+		// When the first listener is added, GetSystemCapability request should go out with subscribe=true
```

Comment:
Based on the change to `retrieveCapability()` I believe this should _not_ send `subscribe=true`, right?

```java
final boolean shouldSubscribe = ((subscribe != null) ? subscribe : isSubscribedToSystemCapability(systemCapabilityType)) && supportsSubscriptions();
request.setSubscribe(shouldSubscribe);
```

Reply:
Ah, you are right, the test actually asserts that `subscribe=false` as you mentioned. The comment is wrong. I will update it.